.gitignore

```diff
@@ -9,6 +9,15 @@ deb/*
 __pycache__
 debian/changelog.*
 pkgs
-misc/mime/[^p]*
-misc/mime/mimeapps.list
-misc/mime/mimeinfo.cache
+/misc/mime/[^p]*
+/misc/applications/mimeapps.list
+/misc/applications/mimeinfo.cache
+/misc/applications/defaults.list
+/qubes-rpc/bin-qfile-unpacker
+/qubes-rpc/qubes-fs-tree-check
+# Created by builder
+/hash
```

Comment:
where?

Reply:
I definitely had the `hash` file in a bunch of repositories.  Maybe an old builder created it?